packages/zoe/src/contractFacet/types.js

```diff
@@ -61,11 +68,11 @@
  * change, overall rights will be unchanged, and a reallocation can
  * only effect offer safety for seats whose allocations change.
  *
- * @param  {SeatStaging} seatStaging
- * @param {SeatStaging} seatStaging
- * @param {SeatStaging=} seatStaging
- * @param {SeatStaging=} seatStaging
- * @param {SeatStaging=} seatStaging
+ * @param {SeatStaging} seatStaging1
+ * @param {SeatStaging} seatStaging2
+ * @param {SeatStaging=} seatStaging3
+ * @param {SeatStaging=} seatStaging4
+ * @param {SeatStaging=} seatStaging5
```

Comment:
This is the only way I know how to do variadic arguments:

```js
/**
 * @typedef {(seatStaging1: SeatStaging, seatStaging2: SeatStaging, ...seatStagingRest: Array<SeatStaging>) => void} Reallocate
 * .... rest of documentation
 */
```

Reply:
That is net better.

Reply:
Thanks, @michaelfig! This works. I'm not sure why a similar thing I was trying a few months ago wasn't working (was erroring in use, wanting an array from callers). Maybe I didn't have seatStaging1 and 2, or maybe it was a callback rather than a typedef. But anyways, thanks!